src/components/Player/AudioInfo/AudioInfoComponent.js

```diff
@@ -0,0 +1,25 @@
+import audioInfoComponentHTML from "./AudioInfoComponent.html";
+
+export class AudioInfoComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+    this.imageSrc =
```

Comment:
move to props - receive from top level Player component

Reply:
why this was not fixed?